src/com/company/M2/Task46.java

```diff
@@ -0,0 +1,40 @@
+package com.company.M2;
+
+public class Task46 {
+    public static String firstWord = "NOTFORYOU";
+    public static String lastWord = "YESNOTFORYOU";
```

Comment:
please use correct naming for constants (`XXX_YYYY`). and do not forget `final` modifier